### PR TITLE
Easy way to access and read sidebars on .compact

### PR DESCRIPTION
**Pull request by angrypotato1**
adds a link to a subreddit's sidebar when viewing a subreddit

also killed some unnecessary whitespace

### CODE REVIEW

File: r2/r2/templates/redditheader.compact [hunk 1]
```diff
@@ -111,6 +111,11 @@
     <div class="menuitem">
       ${plain_link("search", "/search")}
     </div>
+    %if not isinstance(c.site, DefaultSR) and not c.cname:
```
Comment (bsimpson63) [inline r2/r2/templates/redditheader.compact]: Why the `c.cname` check?

`not isinstance(c.site, DefaultSR)` will still create the link on multireddits and other subreddits that don't have sidebars.

**Comment by angrypotato1** [comment]
I did not realise it would include multireddits. Fixed.

File: r2/r2/templates/redditheader.compact [hunk 2]
```diff
@@ -111,6 +111,11 @@
     <div class="menuitem">
       ${plain_link("search", "/search")}
     </div>
+    %if isinstance(c.site, FakeSubreddit):
```
Comment (bsimpson63) [inline r2/r2/templates/redditheader.compact]: should this be `not`?

**Comment by angrypotato1** [comment]
Yes. Oops.

**Comment by bsimpson63** [comment]
:fish: thanks, this looks good

**Comment by angrypotato1** [comment]
awesome, can't wait to see this introduced. Always been a peeve of mine when sidebars aren't accesible

**Comment by bsimpson63** [comment]
Merged as 4a90ec013f08a114ecc69d23d7e82f83f3ce5f34

**Comment by bsimpson63** [comment]
What's your reddit username so I can give you an opensourcer trophy?

**Comment by angrypotato1** [comment]
/u/angrypotato1

Thank you very much for this, just used the feature

**Pull request closed without merging**